misc/multihash/Cargo.toml

```diff
@@ -16,6 +16,8 @@ readme = "README.md"
 documentation = "https://docs.rs/multihash/"
 
 [dependencies]
+blake2 = { version = "0.7", default-features = false }
 sha1 = "0.5"
 sha2 = { version = "0.7", default-features = false }
 tiny-keccak = "1.2"
+unsigned-varint = { version = "0.2", default-features = false }
```

Comment:
Why `default-features = false`?

Reply:
I added that to avoid pulling in `bytes` and `tokio` as indirect dependencies because I thought they were turned on by default in `unsigned-varint`. But upon a [second look](https://github.com/paritytech/unsigned-varint/blob/master/Cargo.toml#L11), they actually aren't, which makes that `default-features = false` useless. That's a good catch; I'll remove it.